frontend/customization/config/footer.json

```diff
@@ -1,25 +1,23 @@
 {
   "socialNetworks": [
-    { "id": "facebook", "url": "https://www.facebook.com/parcnationaldesecrins/" },
-    { "id": "twitter", "url": "https://twitter.com/pnecrins" }
+    { "id": "facebook", "url": "https://www.facebook.com/GeotrekCommunity/" },
+    { "id": "twitter", "url": "https://twitter.com/GeotrekCom" }
   ],
   "links": [
     {
-      "informationID": 4
+      "informationID": 3
     },
-    { "label": "footer.links", "url": "https://www.ecrins-parcnational.fr/liens-internet" },
-    { "label": "footer.access", "url": "https://www.ecrins-parcnational.fr/venir-dans-les-ecrins" },
     {
-      "label": "footer.legalMentions",
```

Comment:
If we remove those translations keys, it will remove them also in the translations files, while they are certainly used by some existing Geotrek-rando portals.

Reply:
I put it back, but I am not sure translations are removed automatically ?

Reply:
Better to keep using these keys I think as they are generic and useful.

Reply:
Sure, I put it back below.